docs/running/cloud/amazon.rst

```diff
@@ -276,6 +289,7 @@ Autoscaling leverages Mesos containers to provide an execution environment for t
 #. Launch the leader node in AWS using the :ref:`launchCluster` command: ::
 
     (venv) $ toil launch-cluster <cluster-name> \
```

Comment:
It looks like our Kubernetes implementation supports scaling up and down through Kubernetes's cluster autoscaler (which you get when you pass a range of numbers instead of a single number for the number of nodes you want of a given type, when launching the cluster.

Running the Toil-integrated autoscaler as part of the workflow needs the `AbstractScalableBatchSystem` functions to e.g. drain nodes to safely scale them away. The Kubernetes cluster autoscaler is supposed to take care of all of that inside of Kubernetes and not inside the individual workflows.